src/radioButtonGroup/index.js

```diff
@@ -13,12 +13,14 @@ class RadioButtonGroup extends React.Component {
     };
   }
 
-  handleCheckListChange = (currentlyActive) => {
-    console.log(currentlyActive);
+  handleCheckListChange = (e, option) => {
+    const { onClick } = this.props;
+    e.stopPropagation();
     this.setState({
-      currentlyActive,
+      currentlyActive: option.label,
     });
-  }
+    if (onClick) { onClick(option); }
```

Comment:
Is it just a callback ? Then remove if case, just directly call onClick fn and set default props for onClick as empty fn.

Reply:
Removed if condition. Default props already added.